### DISCUSSION TITLE
Welcome email points to wrong feedback forum

### DISCUSSION THREAD

**Discussion by mstange** [community/community#17675] [Other Feature Feedback, Questions, & Ideas] [answered]
The "Welcome to the GitHub Code Search Technology Preview!" email says "Please visit the [Code Search Feedback area](https://github.com/github/feedback/discussions/categories/code-search-feedback) to ask questions and share your feedback", linking to the following URL: https://github.com/github/feedback/discussions/categories/code-search-feedback

It looks like this URL is outdated and should be changed to https://github.com/github/feedback/discussions/categories/code-search-and-navigation-feedback instead.

**Comment by look** [comment]
Whoops, good catch! Thanks for the report, we missed updating this when we changed the discussion category to include Code Navigation.

**Comment by colinwm** [comment, marked as answer]
Thanks for reporting this. It's fixed now.
